config/locales/en.yml

```diff
@@ -91,6 +128,14 @@ en:
           unmet_needs_utilities: "Utilities"
           unmet_needs_transportation: "Transportation"
           unmet_needs_other: "Other unmet needs"
+        attestation:
```

Comment:
this looks like it was part of what kristi and i worked on today but the text is a little different than what we added (we thought "Read the the applicant" didn't sound like English so we used a sentence from another part of the page)

Reply:
The keys we've chosen and the nesting are different for attestation as well